packages/jest-util/src/create_process_object.js

```diff
@@ -21,7 +21,6 @@ function createProcessEnv() {
     return deepCyclicCopy(process.env);
   }
 
-  // $FlowFixMe: Apparently Flow does not understand that this is a prototype.
   const proto: Object = Object.getPrototypeOf(process.env);
```

Comment:
Do we still need to annotate this with Object type or will Flow now infer that?

Reply:
Seems like we need it

![image](https://user-images.githubusercontent.com/1404810/45744741-8c516800-bbff-11e8-84f2-cc20782dbd8f.png)